src/Extension.ts

```diff
@@ -70,5 +71,28 @@ export class Extension {
 				}
 			)
 		);
+
+		this.dispose.track(
+			vscode.commands.registerCommand(
+				"hediet.vscode-drawio.newDiagram", async () => {
+					const targetUri = await vscode.window.showSaveDialog({
+						saveLabel: "Create",
+						filters: {
+							"Diagrams": ["drawio"]
+						}
+					});
+					if(!targetUri) { return; }
```

Comment:
Please run prettifier to format your source! ;)

Reply:
This is my first-time with Prettifier. That's Awesome. Finally, I pass the `yarn lint` hehe.